diff --git a/index.js b/index.js
index 67b3e73177e6b7b57eb6d90475c2c9822c4ff65d..bac4a84390f0c8aa4f0fe3fe98a05b5d4ffae695 100755
--- a/index.js
+++ b/index.js
@@ -12,7 +12,7 @@ var pull = function (image, opts, cb) {
   image = parse(image)
   if (!image) throw new Error('Invalid image')
 
-  var request = docker({host: opts.host, version: opts.version || 'v1.21'})
+  var request = docker({host: opts.host, version: opts.version || 'v1.24'})
   var that = new events.EventEmitter()
   var layers = {}
   var progress = {}
